### PR TITLE
API document update. 

### PR DESCRIPTION
**Pull request by cmcm2222**
### Background
As API updated, I apply api which updated. 
`/api/v1/blocks/` and block parameter, proposed_time .. etc. 

But as from @kfangw 's request, I don't add status api description yet. 
Please note this point.

### CODE REVIEW

File: docs/API_v1/transactions.md [hunk 1]
```diff
@@ -6,8 +6,40 @@ Transactions API
 
 
 ### Payment transaction  [POST] 
-//TODO: How to make a transaction and sign 
 
++ Data Body consist of 3 parts, ; T, H, B
```
Comment (Geod24) [inline docs/API_v1/transactions.md]: Would be good to add an example here, with dummy values.

File: docs/API.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ SEBAK, the next BOScoin network with ISAAC consensus protocol.
 <!-- partial(API_v1/models.md) -->
 <!-- partial(API_v1/transactions.md) -->
 
+
```
Comment (Geod24) [inline docs/API.md]: Unneeded

File: docs/API_v1/accounts.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Group Accounts
 Account API
 
-## Account Details [/api/v1/accounts/{address}]
+## Account Details [/api/v1/accounts/{address}]
```
Comment (Geod24) [inline docs/API_v1/accounts.md]: Extra blank space

File: docs/API_v1/models.md [hunk 4]
```diff
@@ -12,8 +12,9 @@
         + templated: true (boolean)
 + address: GDEPYGGALPJ5HENXCNOQJPPDOQMA2YAXPERZ4XEAKVFFJJEVP4ZBK6QI (string, required) - The account’s public key encoded into a base32 string representation.
 + balance: 500000000000 (string) - GON. 1 BOS = 10,000,000 GON
-+ sequence_id: 0 (number) - The Current sequence number. It needed to submitting a transaction from this account
 + linked: "" - linked with freezing account. 
++ sequence_id: 0 (number) - The Current sequence number. It needed to submitting a transaction from this account
```
Comment (Geod24) [inline docs/API_v1/models.md]: Why the change ?

File: docs/API_v1/transactions.md [hunk 5]
```diff
@@ -6,8 +6,40 @@ Transactions API
...
+    + T : ‘transaction’
+
+    + H : H means Header. it consists of version, hash, signature & created.
```
Comment (Geod24) [inline docs/API_v1/transactions.md]: Stylistic comment: "X means Y" is repetitive and not engaging. I usually recommend a shorter style, e.g.:
```
H: Header, consisting of `Version`, `Hash` and `Signature`
 - Version: Currently fixed to 1
 - Hash: sha256(sha256(B)) <== With a link to `B` section here
 - Signature: xdr.Sign(ed25519.Sign(Hash, NetworkID, Seed))
    We use [Stellar's key](https://github.com/stellar/go/tree/master/keypair) for signing.
    We also provide a[JavaScript](https://github.com/bosnet/sebakjs-util) and [Python](https://github.com/spikeekips/sebakpy-util) SDK.

File: docs/API.md (hunk 2 above)
Comment (cmcm2222) [reply docs/API.md]: ??? @Geod24  I think this comment for generating docs. isn't it??

File: docs/API_v1/accounts.md (hunk 3 above)
Comment (cmcm2222) [reply docs/API_v1/accounts.md]: OK.

File: docs/API_v1/transactions.md (hunk 5 above)
Comment (cmcm2222) [reply docs/API_v1/transactions.md]: Oh thanks @Geod24 .
I like your style. 
I will try it. 

but you should generate API doc through this command `https://github.com/bosnet/sebak/tree/master/docs/API_v1`

File: docs/API_v1/transactions.md (hunk 1 above)
Comment (spikeekips) [reply docs/API_v1/transactions.md]: @cmcm2222 This dummy data should be fixed. the `\"payment\"` is wrong, it should be `"type": "payment"`.

**Comment by kfangw** [comment]
* This PR need to be moved to bosnet/docs, not bosnet/sebak
* at the Page section, `reverse` query for prev/next is wrong.

**Comment by cmcm2222** [comment]
@spikeekips  I will do right away.

**Comment by cmcm2222** [comment]
@kfangw @Geod24 @soonkuk  Now I move this PR to [docs repo](https://github.com/bosnet/docs)
